modules/ROOT/pages/microsoft/microsoft-365-ops-connector-reference.adoc

```diff
@@ -26,10 +26,10 @@ Release Notes: xref:release-notes::connector/microsoft-365-ops-connector-release
 |===
 | Name | Type | Description | Default Value | Required
 |Name | String | The name for this configuration. Connectors reference the configuration with this name. | | x
-| Connection a| * <<dynamics-365-for-operations_oauth2, OAuth2>> 
-* <<dynamics-365-for-operations_oauth2-user-password, OAuth Username Password>> 
+| Connection a| * <<dynamics-365-for-operations_oauth2, OAuth2>>
+* <<dynamics-365-for-operations_oauth2-user-password, OAuth Username Password (Deprecated)>>
  | The connection types to provide to this configuration. | | x
-| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances as appropriate. |  | 
+| Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances as appropriate. |  |
```

Comment:
That spreadsheet you started has this wording (which I think is better):
Configures the minimum amount of time that a dynamic configuration instance can remain idle before Mule considers it eligible for expiration. This does not mean that the instance expires at the exact moment that it becomes eligible. Mule purges the instances as appropriate.